src/firefly/js/ui/DownloadDialog.jsx

```diff
@@ -190,10 +190,11 @@ export function DownloadOptionPanel ({groupKey='DownloadDialog', cutoutSize, hel
     const maskPanel = (<BgMaskPanel key={bgKey} componentKey={bgKey}
                                    onMaskComplete={() =>hideDownloadDialog()}/>);
 
-    const saveAsProps = {
+    saveAsProps = {
         initialState: {
             value: get(dlParams, 'BaseFileName')
-        }
+        },
+        ...saveAsProps
     };
```

Comment:
`saveAsProps` should really be done a `slotProps` but since we are up against a deadline it will have to be done at another time.

Reply:
DownloadDialog was designed to be composable.  Several of these 'show' flags to render custom components should not be here, but passed in as `children`.